lib/TableGen/GenDialect.cpp

```diff
@@ -571,7 +571,9 @@ class SymbolTable {
   std::unordered_set<std::string> m_names;
 
 public:
-  std::string chooseName(StringRef name) { return chooseName(ArrayRef(name)); }
+  std::string chooseName(StringRef name) {
+    return chooseName(makeArrayRef(name));
+  }
```

Comment:
Why did you revert this change?

Reply:
I did not revert this intentionally. I just used the GitHub web editor. Maybe the fork being generated is not up-to-date.

Reply:
Sounds like a race-condition in GitHub’s web ui. (I.e. start editing from a certain state, but once one presses save, GitHub gets the new state, which is different when someone pushed in-between.)

Reply:
Yes. Sounds likely, and I rebased onto recent dev.